src/service/url-replacements-impl.js

```diff
@@ -313,6 +313,20 @@ export class GlobalVariableSource extends VariableSource {
       return Date.now();
     });
 
+    //Returns the human readable timestamp in format of 2011-01-01 11:11:11.
```

Comment:
toLocaleString formats timestamp in different ways
for example, `en-US` gives `7/13/2017, 1:47:53 PM`
they will receive all different time formatting at server side :-).

how would the timezone offset look like?